.ecrc

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
.ecrc file is deprecated see https://github.com/editorconfig-checker/editorconfig-checker?tab=readme-ov-file#configuration

Reply:
Seems like this is not yet supported in the latest release - still made the change and passing the config file explicitly.